src/controllers/uploader.ts

```diff
@@ -35,7 +36,7 @@ export const uploadImages = (
       }
       const returnData = {
         signedRequest: data,
-        url: `https://${S3_BUCKET}.s3.amazonaws.com/${fileName}`,
+        url: `https://${S3_BUCKET}/tindev-image.s3.amazonaws.com/${fileName}`,
```

Comment:
I think you need to change the `signedRequest` instead. Since we are using that to upload the image. But I don't think it's a good idea. Check this: https://forums.aws.amazon.com/thread.jspa?threadID=73964

Reply:
Yea, it shows the error, and it also shows the bug when we pass the token as header